enterprise/static/enterprise/enterprise_program_enrollment_page.css

```diff
@@ -0,0 +1,268 @@
+}
+
+.partnered-text {
+    font-size: 16px;
+    color: #414141;
+    margin-top: 5px;
+    margin-bottom: 20px;
+}
+
+.partnered-text strong {
+    font-size: 1.05em;
+}
+
+.program-confirmation-title {
+    font-size: 1.6em;
+    color: #2489c2;
+    font-weight: normal;
+    margin-bottom: 6px;
+}
+
+.program-detail .program-title {
+    font-size: 1.5em;
+    font-weight: bold;
+}
+
+.program-detail .program-org {
+    font-size: 1.2em;
+    font-weight: 600;
+}
+
+.program-detail .program-overview {
+    font-size: 16px;
+    margin: 0.5em 0;
+}
+
+.program-bullet-points {
+    list-style: none;
+    margin: 1em 0;
+}
+
+.program-detail .program-bullet-points .bullet-item {
+    font-size: 16px;
+    margin-left: 20px;
+}
+
+.program-detail .program-bullet-points .bullet-item a {
+    font-size: 16px;
+}
+
+.program-bullet-points li:before {
+    content: '•';
+}
+
+.program-price,
+.program-price * {
+    font-size: 16px;
+    margin: 0;
+}
+
+.final-price {
+    color: #3c763d;
+}
+
+.btn-confirm {
+    background: #0075B4;
+    border: none;
+    color: #fff;
+    border-radius: 5px;
+    margin: 10px 0;
+    padding: 10px 15px;
+    box-shadow: none;
+    font-weight: bold;
+}
+
+.btn-confirm:hover,
+.btn-confirm:focus {
+    cursor: pointer;
+    background: #0b5177;
+}
+
+.program-summary-container {
+    padding-left: 20px;
+}
+
+.program-summary-header {
+    padding: 10px 0;
+}
+
+.program-summary-header strong {
+    font-size: 1.2em;
+    padding-left: 3px;
+}
+
+table.program-summary {
+    border: 1px solid #cccccc;
+    border-radius: 5px;
+    table-layout: fixed;
+    width: 100%;
+}
+
+table.program-summary tr td {
+    text-align: left;
+    vertical-align: top;
+    padding: 15px 0;
+}
+
+table.program-summary td.summary-item {
+    font-size: 16px;
+    font-weight: 600;
+    padding-left: 15px;
+    width: 20%;
+}
+
+.summary-content,
+.summary-content * {
+    font-size: 16px;
+}
+
+.course-item ~ .course-item {
+    padding-top: 10px;
+}
+
+table.program-summary tr.row-separator td {
+    padding: 0 10px;
+}
+
+.row-separator .table-row-separator {
+    width: 100%;
+    border: 1px solid rgba(204, 204, 204, .50);
+}
+
+.text-underline {
+    text-decoration: underline;
+}
+
+#messages .alert {
+    position: relative;
+    padding: 10px 10px 10px 35px;
+    border: 1px solid transparent;
+    box-shadow: none;
+    border-radius: 2px;
+    margin-bottom: 8px;
+}
+
+#messages .alert > .fa {
+    position: absolute;
+    left: 11px;
+    top: 13px;
+    font-size: 16px;
+}
+
+#messages .alert span {
+    display: block;
+}
+
+#messages .success {
+  background-color: #ecfaec;
+  border-color: #b9edb9;
+  color: #3c763d;
+}
+
+#messages .info {
+  background-color: #f2f8fb;
+  border-color: #cce3f0;
+  color: #31708f;
+}
+
+#messages .warning {
+  background-color: #fcf8e3;
+  border-color: #faebcc;
+  color: #8a6d3b;
+}
+
+#messages .error {
+  background-color: #f2dede;
+  border-color: #ebccd1;
+  color: #a94442;
+}
```

Comment:
How much of this is copy/paste from enterprise_course_enrollment_page.css? Is it worth consolidating to a base.css which contains the common styles and can be included on both pages?

Reply:
The `#messages` and `.btn-confirm`, and about the first ~86 lines are copy pasted, with some changes within those lines. The `enterprise_course_enrollment_page.css` contains additional styles that were of course deleted here. I was thinking to combine them, but I was also aware that in the future there would be stories about consolidating the templates with fragments (or something), so I didn't bother too invest effort into doing any consolidation work here.